checkr-official.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 1.9.3'
 
   s.add_dependency('rest-client', '>= 1.4', '< 3.0')
-  s.add_dependency('json', '~> 1.8.1')
+  s.add_dependency('json', '>= 1.8.1')
```

Comment:
Lol, there's not even a rest-client over 2. We should just remove that `< 3.0` constraint.